mayan/apps/documents/models/metric.py

```diff
@@ -0,0 +1,6 @@
+from mayan.apps.documents.models.document_file_models import DocumentFile
+from django.db import models
+
+class Metric(models.Model):
+    """Model that describes a metric"""
+    metric_name = models.TextField()
```

Comment:
there should also be a field that stores the type of the metric from these options https://docs.djangoproject.com/en/3.2/ref/forms/fields/#built-in-field-classes. I guess this can also be a text field. That would later be reprocessed as a class name.

Reply:
Using the field classes will make it much easier to implement the form/view correctly